contrib/fwupd.spec.in

```diff
@@ -311,10 +311,13 @@ mkdir -p --mode=0700 $RPM_BUILD_ROOT%{_localstatedir}/lib/fwupd/gnupg
 # workaround for https://bugzilla.redhat.com/show_bug.cgi?id=1757948
 mkdir -p $RPM_BUILD_ROOT%{_localstatedir}/cache/fwupd
 
+# Remove "disable" preset as it is not needed on Fedora (disabled by default)
+rm $RPM_BUILD_ROOT%{_presetdir}/50-fwupd-refresh.preset
+
```

Comment:
What about other RPM distros?  Does RHEL do this?  Does OpenSUSE?

Reply:
With the Debian package removing the disable preset config in https://github.com/fwupd/fwupd/blob/main/contrib/debian/rules#L80, I'm wondering who wants this config file. Maybe we could make it opt-in and add a meson build option instead.

Reply:
I think very few people actually change the default config options in reality. Maybe remove it completely? @superm1

Reply:
Yeah I think axe it completely.